app/helpers/application_helper.rb

```diff
@@ -82,4 +82,10 @@ def find_version_author_email(version)
       ''
     end
   end
+
+  def nationality_name(nationality)
+    return '' if nationality.blank?
+    c = ISO3166::Country[nationality]
+    c.translations[I18n.locale.to_s] || c.name
```

Comment:
`locale` is available straight and this could do the same:
```ruby
ISO3166::Country(nationality).translations[locale.to_s] || ISO3166::Country(nationality).name
```

but in the end same functionality... not important!